runtime/metrics.rs

```diff
@@ -138,29 +138,29 @@ pub fn metrics_op(name: &'static str, op_fn: Box<OpFn>) -> Box<OpFn> {
 
     match op {
       Op::Sync(buf) => {
-        metrics.op_sync(bytes_sent_control, bytes_sent_data, buf.len());
+        metrics.op_sync(bytes_sent_control, bytes_sent_data, 0);
```

Comment:
A bit unlucky that we lose this information, it will be a breaking change to the metrics interface

Reply:
Changed the unit tests to expect `0`, we reached a consensus on Discord that `0` was better than an estimate or `8B/op`.